src/js/state.js

```diff
@@ -0,0 +1,77 @@
+class State {
+	/**
+	* Class constructor.
+	* @param {RadioNodeList} [inputs] - A NodeList of radio input elements
+	*/
+	constructor(inputs) {
+		this.inputs = inputs;
+		this.parent = inputs.length > 1 ? this.inputs[0].closest('.o-forms-input') : null;
+
+		this._verify();
+
+		this.stateEl = null;
```

Comment:
It's maybe more intuitive to set this to the element immediately in the constructor? Especially as a public property.